.github/workflows/on-push-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Run tests
+on: push
```

Comment:
In VSF we are running our jobs on `push` and `pull_request` events. Are you sure you want to run them only on `push`?

Reply:
We didn't do this before, but now we can of course :)